Code/Framework/AzManipulatorTestFramework/Include/AzManipulatorTestFramework/ActionDispatcher.h

```diff
@@ -79,89 +79,104 @@ namespace AzManipulatorTestFramework
         virtual void SetSelectedEntityImpl(AZ::EntityId entity) = 0;
         virtual void SetSelectedEntitiesImpl(const AzToolsFramework::EntityIdList& entities) = 0;
         virtual void EnterComponentModeImpl(const AZ::Uuid& uuid) = 0;
-        template <typename... Args>
-        void Log(const char* format, const Args&... args);
+        void Log(const char* format, ... );
         bool m_logging = false;
+
     private:
         const char* KeyboardModifierString(const AzToolsFramework::ViewportInteraction::KeyboardModifier& keyModifier);
     };
 
-    template <typename DerivedDispatcherT>
-    template <typename... Args>
-    void ActionDispatcher<DerivedDispatcherT>::Log(const char* format, const Args&... args)
+    template<typename DerivedDispatcherT>
+    void ActionDispatcher<DerivedDispatcherT>::Log(const char* format, ...)
     {
+        va_list args;
+        va_start(args, format);
+
         if (m_logging)
         {
-            AZ_Printf("ActionDispatcher", format, args...);
+            vprintf(format, args);
+            printf("%s", "\n");
```

Comment:
Ah yes sorry I mentioned this briefly in the description. I think because of the amount of spam that was happening in integration tests, `AZ_Printf` was disabled during runs (so nothing gets printed to the console). This however is super annoying for debugging with the manipulator test framework (when you set logging to true). I asked @lumberyard-employee-dm about this and he suggested just using the standard `printf` call, so now you can see the log/trace output when debugging tests (this should never by left 'on' when merged).

Reply:
Ah ok sorry, I missed the description. Disabling AZ_Printf sounds pretty dumb to me, because now everyone will switch to printf and the spam will gradually build up again. I think there was a proposal at some point to have more granular verbosity settings, not sure what happened to that.

Reply:
Could of questions:
- I was wondering if `printf(format, args...)` doens't work? It would have avoided to have to change the function templates.
- Why using `printf("%s", "\n");` instead of just `printf("\n");`?

Reply:
I believe it's because its more safe. In this case it doesn't matter too much, but if you are doing printf(input) and then someone modifies so input has "%s", you can get a buffer overrun and print application memory

Reply:
I get the case from `printf(input)`, but I don't see any particular benefit in `printf("%s", "\n");`

Reply:
It's just habit I guess as it's generally the preferred approach for `printf` whenever it's used so I just default to that, I can change it if absolutely necessary but I don't think it's really hurting much